scripts/azureml-assets/azureml/assets/validate_assets.py

```diff
@@ -574,6 +595,188 @@ def validate_model_assets(latest_asset_config: assets.AssetConfig, validated_ass
         return 1
 
 
+def validate_model_spec(asset_config: assets.AssetConfig, validated_model_map: dict):
+    """Validate model spec."""

```

Comment:
Could you also add info about the args and return value in the comment please, similar to https://github.com/Azure/azureml-assets/blob/b9a80fd47e2bda0dd37f4666c49b2811cf7e4b08/scripts/azureml-assets/azureml/assets/validate_assets.py#L175C1-L182C8 ?